app.js

```diff
@@ -13,13 +13,18 @@ const handlebars = require('handlebars');
 const layouts = require('handlebars-layouts');
 const handlebarsWax = require('handlebars-wax');
 const Sentry = require('@sentry/node');
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
`Configuration` is really generic and difficult to understand. I have no clue what this package is doing.
After reading the npm package description I understand what it should do, but I still think the name should be changed here.
There is also no documentation at all, how to use and configure that package.